time/src/main/java/io/spine/time/Constants.java

```diff
@@ -25,15 +25,24 @@
  *
  * @author Mykhailo Drachuk
  */
-class SiTime {
+class Constants {
```

Comment:
It's a question for me why we even define these constants and not using something like `java.util.concurrent.TimeUnit` (since 1.5) for conversion.

My guess was that these values were to be re-used outside the library. But now I realize the class is private to this package.